pkg/postgres/schema/deployments.go

```diff
@@ -106,7 +106,7 @@ type Deployments struct {
 	ClusterId                     string                  `gorm:"column:clusterid;type:varchar"`
 	ClusterName                   string                  `gorm:"column:clustername;type:varchar"`
 	Annotations                   map[string]string       `gorm:"column:annotations;type:jsonb"`
-	Priority                      int64                   `gorm:"column:priority;type:integer"`
+	Priority                      int64                   `gorm:"column:priority;type:bigint"`
```

Comment:
This feels like a large type for a Priority field.  But I'm not an expert on the data.  If that is something we should look as to whether some of the fields are right sized, that should be done separately.